src/envs/behavior.py

```diff
@@ -1,62 +1,145 @@
 """Behavior (iGibson) environment."""
-# pylint: disable=import-error
+# pylint: disable=import-error,ungrouped-imports
 
 import functools
 import itertools
 import os
 from typing import List, Set, Optional, Dict, Callable, Sequence, Iterator
 import numpy as np
+from numpy.random._generator import Generator
+
 try:
     import bddl
     import igibson
+    from igibson import object_states
     from igibson.envs import behavior_env
-    from igibson.objects.articulated_object import ArticulatedObject  # pylint: disable=unused-import
+    from igibson.objects.articulated_object import (  # pylint: disable=unused-import
+        ArticulatedObject, )
     from igibson.objects.articulated_object import URDFObject
     from igibson.object_states.on_floor import RoomFloor
     from igibson.robots.behavior_robot import BRBody
-    from igibson.utils.checkpoint_utils import \
-        save_internal_states, load_internal_states
-    from igibson.activity.bddl_backend import SUPPORTED_PREDICATES, \
-        ObjectStateUnaryPredicate, ObjectStateBinaryPredicate
+    from igibson.activity.bddl_backend import SUPPORTED_PREDICATES,\
+        ObjectStateUnaryPredicate,ObjectStateBinaryPredicate
+    from predicators.src.envs.behavior_options import navigate_to_obj_pos,\
```

Comment:
The reason is so that we don't have to make installing igibson and behavior necessary for predicators (if we did, then Ashay and Amber for example would have to install and setup igibson to work on their projects, which have nothing to do with BEHAVIOR atm)

Reply:
I understand why we need the `from igibson` and `from bddl` imports in the `try: except`. This comment was regarding the `from predicators.src.envs.behavior_options import ...` line, which should be able to be moved outside?

Reply:
i see, yes, good point!